components/property_info.rst

```diff
@@ -0,0 +1,295 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertInfo
+
+The PropertyInfo Component
+==========================
+
+    The PropertyInfo component extracts information about PHP class' properties
+    using metadata of popular sources.
+
+The PropertyInfo component is able to extract the following information:
+
+* List of properties exposed by a class
+* Types of a property
+* It's short and long DockBlock description
+* If the property is readable or writable
+
+To do so, the component use extractors. It natively support the following
+metadata sources:
+
+* ``ReflectionExtractor``: use the PHP Reflection API (setter type hint,
+  return and scalar type hint for PHP 7+, accessor methods)
+* ``PhpDocExtractor``: use the PHPDoc of properties and accessor methods
+* ``DoctrineExtractor``: use metadata provided by the Doctrine ORM
+* ``SerializerExtractor``: use groups metadata of the Serializer component
+
+Custom extractors can be also be registered.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/property-info``
+  on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/PropertyInfo).
+
+.. include:: /components/require_autoload.rst.inc
+
+To use the :class:`Symfony\\Component\\PropertyInfo\\Extractor\\PhpDocExtractor`,
+install `phpDocumentator Reflection`_.
+To use the :class:`Symfony\\Component\\PropertyInfo\\Extractor\\SerializerExtractor`
+extractor, install the :doc:`Serializer component </components/serializer>`.
+To use the :class:`Symfony\\Bridge\\Doctrine\\PropertyInfo\\DoctrineExtractor`,
```

Comment:
this will be rendered as one paragraph with 3 sentences, all looking like "To use the ...Extractor extractor, install the ... component", which I find a not so nice to read paragraph (it breaks to read flow).

A common practice is to document additional dependencies in the related sections in a note directive. For instance, in the "DoctrineExtractor" section, put something like:
```rst
.. note::

    You must install the Doctrine Bridge and the Doctrine ORM to be able
    to use this extractor. You can install them by running
    ``composer require doctrine/orm symfony/doctrine-bridge``.
```